PacketDefinitions420/PacketNotifier.cs

```diff
@@ -3736,15 +3736,20 @@ public void NotifyUpdatedStats(IAttackableUnit u, int userId = 0, bool partial =
         {
```

Comment:
Update the function's name to `NotifyOnReplication` and rearrange it in alphabetical order

Reply:
:heavy_check_mark: